app/src/main/res/values-zh-rCN/arrays.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="color_theme_entries">
+        <item>正常</item>
+        <item>高对比度</item>
+    </string-array>
+    <string-array name="screen_orientation_entries">
+        <item>自动横屏</item>
+        <item>锁定了横屏</item>
+        <item>撤销了锁定</item>
```

Comment:
Thank you for providing the translations.
I checked a few translations with Google translate and this one translates to "Unlocked". The original string is "Locked Landscape Reversed". The meaning of this option is that the screen is locked in landscape but is in opposite direction (upside-down) to the other "Locked landscape".
Is this a mistake in the translation?